src/js/iconpicker.js

```diff
@@ -931,4 +939,5 @@
             'fa-y-combinator-square', 'fa-yahoo', 'fa-yc', 'fa-yc-square', 'fa-yelp', 'fa-yen', 'fa-yoast', 'fa-youtube',
             'fa-youtube-play', 'fa-youtube-square'
         ];
+        Iconpicker.fontAwesome5Icons = //###REPLACE-WITH-FONT-AWESOME-5-FONTS###
```

Comment:
I expected Icons to be automatically updated wrt the original Repository (Hence, referred to version in Master). So from next upgrade, the only thing required would be to update the URL, until the yml file changes, which would also require minor changes.
Meaning, it would be possible to support multiple versions without much effort.